yal/mainmenu.py

```diff
@@ -58,6 +58,15 @@ async def get_suggested_choices(self, parent_topic_links={}):
 
         return suggested_choices
 
+    async def get_first_time_user(self, whatsapp_id):
+        first_time_user = self.user.metadata.get("first_time_on_main_menu")
+        error = await rapidpro.update_profile(
+            whatsapp_id, {"first_time_on_main_menu": ""}, self.user.metadata
+        )
+        if error:
+            return await self.go_to_state("state_error")
+        return bool(first_time_user)
```

Comment:
What if we do this the other way around, then it will also work for existing users? eg. if `seen_privacy_policy` is blank, then we send the extra message, and set `seen_privacy_polity` to `TRUE`, so that we won't send it again.